maintenance/tables.sql

```diff
@@ -33,84 +33,6 @@
 -- table prefix if any when running these scripts.
 --
 
-
---
--- The user table contains basic account information,
--- authentication keys, etc.
---
--- Some multi-wiki sites may share a single central user table
--- between separate wikis using the $wgSharedDB setting.
---
--- Note that when a external authentication plugin is used,
--- user table entries still need to be created to store
--- preferences and to key tracking information in the other
--- tables.
---
-CREATE TABLE /*_*/user (
```

Comment:
Let's move these table definitions to a separate file and make sure they are included by https://github.com/Wikia/app/blob/dev/includes/wikia/tests/core/WikiaDatabaseTest.php initialization

Reply:
Done in 4748d6d